master/cluster.go

```diff
@@ -2576,9 +2574,9 @@ func (c *Cluster) addDataPartitionRaftMember(dp *DataPartition, addPeer proto.Pe
 	}
 
 	log.LogInfof("action[addDataPartitionRaftMember] try host [%v] to [%v] peers [%v] to [%v]",
-		dp.Hosts, newHosts, dp.Peers, newPeers)
+		dp.Hosts, dp.Hosts, dp.Peers, dp.Peers)
 
-	if err = dp.update("addDataPartitionRaftMember", dp.VolName, newPeers, newHosts, c); err != nil {
+	if err = dp.update("addDataPartitionRaftMember", dp.VolName, dp.Peers, dp.Hosts, c); err != nil {
```

Comment:
`MUST` restore hosts and peers when updating failed.

Reply:
Thanks for your correction, I have modified this problem as you said.

Reply:
> `MUST` restore hosts and peers when updating failed.

Thanks for your correction, I have modified this problem as you said.